.travis.yml

```diff
@@ -1,38 +1,48 @@
 dist: bionic
 language: node_js
-node_js:
- - 14
- - 16
-os:
-  - linux
-  - osx
-  - windows
 
 before_install:
 #  - npm i -g npm@latest
   # install dependency for circom
-  - |
-    if [ $TRAVIS_OS_NAME != "osx" ]; then
-      # build circom from source bcoz the official binary was built with a higher version of libc6(2.8) than the one in CI env(2.7).
-      curl --proto '=https' --tlsv1.2 https://sh.rustup.rs -sSf | sh -s -- -y
-      git clone https://github.com/iden3/circom.git
-      cd circom
-      $HOME/.cargo/bin/cargo build --release
-      $HOME/.cargo/bin/cargo install --path circom
-      cd ..
-      if [ $TRAVIS_OS_NAME = "windows" ]; then
-        export PATH=$PATH:$HOME/.cargo/bin
-      else
-        source $HOME/.cargo/env
-      fi
-    fi
+  # - |
+  #   if [ $TRAVIS_OS_NAME != "osx" ]; then
+  #     # build circom from source bcoz the official binary was built with a higher version of libc6(2.8) than the one in CI env(2.7).
+  #     curl --proto '=https' --tlsv1.2 https://sh.rustup.rs -sSf | sh -s -- -y
+  #     git clone https://github.com/iden3/circom.git
+  #     cd circom
+  #     $HOME/.cargo/bin/cargo build --release
+  #     $HOME/.cargo/bin/cargo install --path circom
+  #     cd ..
+  #     if [ $TRAVIS_OS_NAME = "windows" ]; then
+  #       export PATH=$PATH:$HOME/.cargo/bin
+  #     else
+  #       source $HOME/.cargo/env
+  #     fi
+  #   fi
 
-install:
-
-script:
-  - npx scryptlib download
-  - |
-    if [ $TRAVIS_OS_NAME = "osx" ]; then
-      npm run download:circom
-    fi
-  - npm t
\ No newline at end of file
+jobs:
+  include:
+    - if: branch = master AND type = push

```

Comment:
Will this run in a PR not yet merged to `master`?

Reply:
yes